cpp/src/arrow/builder.h

```diff
@@ -370,6 +372,11 @@ class ARROW_EXPORT NumericBuilder : public PrimitiveBuilder<T> {
           ARROW_MEMORY_POOL_DEFAULT)
       : PrimitiveBuilder<T1>(TypeTraits<T1>::type_singleton(), pool) {}
 
+  using ArrayBuilder::AppendToBitmap;
+  using ArrayBuilder::SetNotNull;
+  using ArrayBuilder::UnsafeAppendNull;
+  using ArrayBuilder::UnsafeAppendToBitmap;
```

Comment:
Can you make only UnsafeAppendNull public? It looks like there's one place where `UnsafeAppendToBitmap(false)` will have to be replaced with `UnsafeAppendNull`